src/openms/source/APPLICATIONS/ConsoleUtils.cpp

```diff
@@ -121,6 +120,7 @@ namespace OpenMS
         {
           LOG_DEBUG << "output shaping: stty size command failed." << std::endl;
         }
+        pclose(fp);
```

Comment:
I also looked at that one before but I was not sure what pclose would do if `fp` is actually `NULL`. 

See http://linux.die.net/man/3/pclose there is no mention of this case here but I also think that if popen actually returns NULL then a call to pclose on that is going to be the least of your problems. The other option would be to put it at line 117 in the branch where `fp != NULL`